contractcourt/briefcase.go

```diff
@@ -533,7 +535,9 @@ func (b *boltArbitratorLog) FetchUnresolvedContracts() ([]ContractResolver, erro
 // swapped out, or resolved.
 //
 // NOTE: Part of the ContractResolver interface.
-func (b *boltArbitratorLog) InsertUnresolvedContracts(resolvers ...ContractResolver) error {
+func (b *boltArbitratorLog) InsertUnresolvedContracts(resolvers []ContractResolver,
+	closures ...func(tx kvdb.RwTx) error) error {
```

Comment:
As this function doesn't know what the closures are anyway, it could maybe be just a single function pointer. Do the loop inside that function